async_service/base.py

```diff
@@ -334,6 +330,5 @@ async def _run_and_manage_task(self, task: TaskAPI) -> None:
         else:
             if task.parent is None:
                 self._root_tasks.remove(task)
-            self.logger.debug("%s: task %s exited cleanly.", self, task)
```

Comment:
what do you think about placing these behind a library level DEBUG flag of some sort that can be enabled/disabled with an environment variable.  These messages, while extremely noisy, are also really useful for debugging "what happened".